driver-core/src/main/java/com/datastax/driver/core/ResultSet.java

```diff
@@ -39,7 +34,11 @@
  * <p/>
  * Note that this class is not thread-safe.
  */
-public interface ResultSet extends Iterable<Row> {
+public interface ResultSet extends PagingIterable<ResultSet, Row> {
+
+    // redeclared only to make clirr happy
+    @Override
+    Row one();
```

Comment:
You mean, you would prefer that I add the exception to `clirr-ignores.xml`? the exact error is 

```
7002: com.datastax.driver.core.ResultSet: Method 'public com.datastax.driver.core.Row one()' has been removed
```